ls1intum/Artemis
# Update server and client dependencies

This PR updates some server and client dependencies

### Checklist

### Steps for Testing
Click through the most important functionalities and check everything is still working
